ecl/agentexec/agentexec.cpp

```diff
@@ -129,8 +115,16 @@ int CEclAgentExecutionServer::run()
 
     try 
     {
-        while (started)
+        while (true)
         {
+#ifdef _CONTAINERIZED
+            if (!pool->waitAvailable(10000))
+            {
+                  if (config->getPropInt("@traceLevel", 0) > 2)
+                      DBGLOG("Blocked for 10 seconds waiting for an available agent slot");
```

Comment:
This will potentially add quite a lot of tracing - especially if the limit is set to 1, which may be common.  Should scale the timing, suppress it if only a single item, and/or think about when this should be reported.

Reply:
traceLevel 3 is considered quite high...

I'll add a jira to revisit whether it's generating too much

Reply:
ok, I missed that.  As long as it doesn't come out in the default tracing.